parsl/executors/status_handling.py

```diff
@@ -155,9 +152,6 @@ def set_bad_state_and_fail_all(self, exception: Exception):
     def status(self):
         return {}
 
-    def create_monitoring_info(self, status: Dict[object, JobStatus]) -> List[object]:
-        return []
-
```

Comment:
These implementations in this file might be necessary (or move them into the provider base class rather than making the base class method abstract) because not all concrete executors implement this method.

Reply:
Yea, you are right. I realized that via CI again. I added these back.